docs/how-to-upload-artifact.md

```diff
@@ -1,5 +1,7 @@
 # How To Push an Operator Artifact To an App Registry
 
+**THIS DOC AND THE SCRIPTS MENTIONED HERE HAVE BEEN DEPRECATED BY IN FAVOR OF A [NEW METHOD](https://github.com/operator-framework/community-operators/blob/master/README.md)**
```

Comment:
why not just delete the file then?

Reply:
@tkashem says there is useful information for developers in here that he wants to retain. If that is not the case I am fine with deleting it.

Reply:
Personally I think that, if there is stuff in this we want to retain it should be moved out of this repo and into courier's docs.

Reply:
@kevinrizza done